SwiftUIHW/SwiftUIHW/AccountDetailView/AccountDetailView.swift

```diff
@@ -0,0 +1,368 @@
+    }
+
+    private var datePickerView: some View {
+        HStack(spacing: 70) {
+            VStack {
+                Picker(Constants.emptyString, selection: $viewModel.month) {
+                    ForEach(1 ..< 12) { mnt in
+                        Text("\(mnt)")
+                    }
+                }
+                .tint(.black)
+                separatorView(width: 100)
+            }
+            VStack {
+                Picker(Constants.emptyString, selection: $viewModel.year) {
+                    ForEach(2010 ..< 2051) { year in
+                        Text("\(year.formatted(.currency(code: Constants.emptyString)))")
+                    }
+                }
+                .tint(.black)
+                separatorView(width: 100)
+            }
+        }
+    }
+
+    private var addButton: some View {
+        Button(Constants.addNowText) {
+            viewModel.addNowButtonAction()
+        }
+        .largeButton()
+        .alert(isPresented: $viewModel.isCvcAlertShown) {
+            Alert(title: Text(Constants.cvcAlertText))
+        }
+    }
+    
+    private var successAlertView: some View {
+        VStack {
+            ZStack {
+                
+                RoundedRectangle(cornerRadius: 15)
+                    .frame(width: 100, height: 50)
+                    .foregroundColor(.lightGray)
+                
+                Text(Constants.successText)
+            }
+        }
+        .transition(.slide)
+    }
+
+    private var backsideCardView: some View {
+        ZStack {
+            
+            RoundedRectangle(cornerRadius: 15)
+                .frame(width: 350, height: 230)
+                .foregroundColor(.clear)
+                .background(LinearGradient(
+                    colors: [.startGradient, .endGradient],
+                    startPoint: .topLeading,
+                    endPoint: .bottomTrailing
+                ))
+
+            VStack(spacing: 20) {
+                Rectangle()
+                    .frame(width: 350, height: 60)
+
+                ZStack {
+                    
+                    Rectangle()
+                        .frame(width: 350, height: 40)
+                        .foregroundColor(.white)
+                    
+                    HStack {
+                        VStack {
+                            separatorView(width: 250)
+                            separatorView(width: 250)
+                            separatorView(width: 250)
+                        }
+                        Spacer()
+
+                        Text("\(viewModel.cvc)")
+                            .padding(.trailing, 55)
+                    }
+                    .padding(.leading, 20)
+                }
+            }
+            .offset(y: -25)
+        }
+        .rotation3DEffect(Angle(degrees: viewModel.frontDegree), axis: (x: 0, y: 1, z: 0))
+        .onTapGesture {
+            viewModel.flipCardAction()
+        }
+    }
+
+    private var cardView: some View {
```

Comment:
```suggestion
    private var cardView: some View {
```
Большая вью или выносим в отдельное View или дробим на вычисляемое свойство